lib/Yam/Agama/Pom/RebootPage.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles installation reboot screen.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::Agama::Pom::RebootPage;
+use strict;
+use warnings;
+
+use testapi;
+
+sub new {
+    my ($class, $args) = @_;
+    return bless {
+        screen => 'agama-install-finished',
```

Comment:
s/screen/tag_installation_complete/ or something starting by `tag_` would do to follow some convention.

Reply:
Solved 257734447a4af251e62c1d1826ecd73a7b8f262a